### PR TITLE
Features/controllers spec

### PR DESCRIPTION
**Pull request by oguntayomicheal**
#### IN this PR, the following task were implemented:
- Add spec for user_controller actions
- Add spec for post_controller actions

### CODE REVIEW

File: spec/requests/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      get '/users/45/posts/45'
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes the correct placeholder text' do
+      get '/users/45/posts/45'
+      expect(response.body).to include('Here is a particular post by a specific user')
+    end
+  end
+end
```
Comment (abdulhamiid) [inline spec/requests/post_spec.rb:37]: Test Result 
image

Please, make sure that the spec file includes tests that ensure the correct view template is rendered 🛠

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @oguntayomicheal 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmetbozaci** [comment]
### Hello @oguntayomicheal 
### _This is an additional review as a result of the second opinion request._ 


- All tests are pass
image



___
_**Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away**_



### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by oguntayomicheal** [comment]
Hello @ahmetbozaci, thanks for the additional review.

File: spec/requests/post_spec.rb (hunk 1 above)
Comment (oguntayomicheal) [reply spec/requests/post_spec.rb:37]: Hi @abdulhamiid, all the test are passing from my end.

**Pull request merged**